src/component/getThought/getThought.css

```diff
@@ -0,0 +1,40 @@
+.old-thought-container{
```

Comment:
Need to add  " word-wrap: break-word;". Otherwise it will show like this in screen shoot.
![image](https://github.com/Technigo/project-happy-thoughts-vite/assets/150081337/986267d6-ae4c-4f71-b56f-9eccc9cf3de2)